From 9b84925f68d29844116d29ad421acb03176b15da Mon Sep 17 00:00:00 2001
From: Oliver Neukum <oneukum@suse.com>
Date: Tue, 2 May 2023 18:10:54 +0200
Subject: [PATCH] xhci: hide include of iommu.h
References: git-fixes
Patch-mainline: Never, kABI fixup

The include confuses the kABI checker by making symbols defined

Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/host/xhci.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/usb/host/xhci.c b/drivers/usb/host/xhci.c
index de4d25557..226883fd9 100644
--- a/drivers/usb/host/xhci.c
+++ b/drivers/usb/host/xhci.c
@@ -21,7 +21,9 @@
  */
 
 #include <linux/pci.h>
+#ifndef __GENKSYMS__
 #include <linux/iommu.h>
+#endif
 #include <linux/iopoll.h>
 #include <linux/irq.h>
 #include <linux/log2.h>
-- 
2.40.1

